### DISCUSSION TITLE
希望打字页面可以直接录音的功能

### DISCUSSION THREAD

**Discussion by yankedi** [WatsonYao/nian_app#85] [Q&A] [answered]
您好，希望在打字页面添加一个直接接录音的选项，虽然有添加附件的选项，但是这个过程未免太割裂，要先录音，再导入，有时候太累了，像我这种高三牲晚上不想打字，就想睡觉前说说话总结一下今天什么的，直接可以录的话方便不少。
当然如果很难做，或者只有很少的人需要的话，不加也是可以的。感谢您一直以来开源付出。

**Comment by WatsonYao** [comment, marked as answer]
感谢反馈，没办法跳转第三方录音app，内置录音功能的话开发成本大，收益小，用到的人很少。
现在语音转文字更方便快捷，也更容易搜索。录音文件大，不方便搜索，也不方便存储和转移。
nian 没有开源，只是可永久免费使用。
